ansible_image_validation/validation-playbooks/per-vm-validation.yaml

```diff
@@ -157,7 +157,7 @@
         line: "\n Accelerated Networking Validation failed since unmanaged drivers are not present."
         create: yes
         state: present 
-      when: ("driver:mlx4_core;driver:mlx5_core" not in check_network_config.stdout_lines)    
+      when: ("unmanaged-devices=driver:mlx4_core;driver:mlx5_core" not in check_network_config.stdout_lines)    
```

Comment:
Can you provide more context into this change? 
1. Previously, what was the intent of this acn validation? 
2. Was this validation introduced with a bug? Why is it failing now? 
3. Is it failing for all RHEL9 images now?

Reply:
@simratsatia 
1.we have added this validation because we have added the script to enable accelerated networking in rhel 9 images.
2. Well this check has been overlooked considering that only half string is needed for validation.
3.This task is failing for all the rhel 9 images because it is not considering only the half part of string but needs full string for validation.